docs/data/pages.ts

```diff
@@ -1,5 +1,7 @@
 import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
 
+const isPreview = process.env.NODE_ENV === 'development';
```

Comment:
Maybe we could add a runtime check to include the path allowing the docs previews in PR deployments? 🤔

Reply:
I finally found an env variable that could do the job
https://docs.netlify.com/configure-builds/environment-variables/#build-metadata

Reply:
Success! 🥳 ❤️

Reply:
Having them in PR deployment would be great indeed :ok_hand:

Reply:
🙌 
The only issue is that the icon is missing for charts. 😢 
![Screenshot 2023-03-22 at 15 33 25](https://user-images.githubusercontent.com/4941090/226920656-97646637-ec6e-416c-9584-bd5c4f346bc4.png)